app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java

```diff
@@ -2903,7 +2903,7 @@ public double getPV3() {
     public double getPCS() {
         return this.dac.getVoltage("PCS");
     }
-
+    
```

Comment:
Shall we revert this change? Please don't modify other files which you are not addressing in a PR.